FrontEnd/src/pages/AboutUsPage/AboutUs.module.css

```diff
@@ -0,0 +1,126 @@
+
+.about-us-section {
+    min-width: 100vw;
+    background: var(--about-us-background-color);
+    justify-content: center; 
+    align-items: center;
```

Comment:
There are redundant justify-content and align-items properties that have no effect.